create-sbuild-chroot-schroot.py

```diff
@@ -90,6 +90,10 @@ def get_sbuild_architecture():
 parser.add_argument("distribution", type=str, nargs="*")
 args = parser.parse_args()
 
+if not os.path.exists(args.config):
+    parser.error(f"config path {args.config} does not exist")
+
+logging.info("Reading configuration")
 with open(args.config) as f:
```

Comment:
Ask for forgiveness, not permission.

I.e. just handle the exception if the file doesn't exist.

Reply:
I mean something like this:

```
try:
    with open(args.config) as f:
         config = read_config(f)
except FileNotFoundError:
    parser.error(f"config path {args.config} does not exist")
```

Actually, better yet would be to use the file type argument for argparse: https://stackoverflow.com/questions/18862836/how-to-open-file-using-argparse

Reply:
Looks like using file type argument will not work in Python 3.14: 

> Deprecated the [argparse.FileType](https://docs.python.org/3.14/library/argparse.html#argparse.FileType) type converter. Anything with resource management should be done downstream after the arguments are parsed. (Contributed by Serhiy Storchaka in [gh-58032](https://github.com/python/cpython/issues/58032).)

https://docs.python.org/3.14/whatsnew/3.14.html#deprecated